cmd/openshift-install/gather.go

```diff
@@ -95,30 +96,30 @@ func runGatherBootstrapCmd(directory string) error {
 	if err != nil {
 		if err2, ok := err.(errUnSupportedGatherPlatform); ok {
 			logrus.Error(err2)
-			return unSupportedPlatformGather()
+			return unSupportedPlatformGather(directory)
 		}
 		return errors.Wrapf(err, "failed to get bootstrap and control plane host addresses from %q", tfStateFilePath)
 	}
 
-	logGatherBootstrap(bootstrap, port, masters)
+	logGatherBootstrap(bootstrap, port, masters, directory)
 	return nil
 }
 
-func logGatherBootstrap(bootstrap string, port int, masters []string) {
-	var (
-		sshport string
-		scpport string
-	)
-	if port != 22 {
-		sshport = fmt.Sprintf(" -p %d", port)
-		scpport = fmt.Sprintf(" -P %d", port)
+func logGatherBootstrap(bootstrap string, port int, masters []string, directory string) {
+	logrus.Info("Pulling logs from bootstrap for debugging")
+
+	client, err := ssh.NewClient("core", fmt.Sprintf("%s:%d", bootstrap, port), nil)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+	if err := ssh.Run(client, fmt.Sprintf("/usr/local/bin/installer-gather.sh %s", strings.Join(masters, " "))); err != nil {
```

Comment:
I'd rather rely on this being in the `PATH` (seems more DRY), but I'm ok with the absolute path (also seems unlikely to change ;) if you think relying on `PATH` is too risky.

Reply:
there's no harm with being absolute. gonna keep it as-is.